### PR TITLE
pull request comment

### PR DESCRIPTION
**Pull request by ethan-gray-01**
just left a comment

### CODE REVIEW

**Comment by ethan-gray-01** [comment]
Intend to add R value onto graphical representation

File: code/ui.R [hunk 1]
```diff
@@ -4,7 +4,7 @@ background-color: #D4D4D4 !important;
 color: #333 !important;
 cursor: not-allowed !important;
 }"
-
+#comment for pull request
```
Comment (zohaib-a-ahmed) [inline code/ui.R:7]: Try not to leave comments like this because as changes are made, this will be deprecated info.

**Review by zohaib-a-ahmed — COMMENTED** [review]
Sloppy work, for next change work on doing better!

**Comment by kate-holdener** [comment]
Please avoid adding unnecessary comments to the source code.

**Pull request closed without merging**